packages/components/src/components/error-modal/error-modal.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import DesktopWrapper from '../desktop-wrapper';
+import MobileWrapper from '../mobile-wrapper';
+import Modal from '../modal';
+import MobileDialog from '../mobile-dialog';
+import ErrorModalContent from './error-modal-content';
```

Comment:
@hamza-deriv please sort imports alphabetically
```suggestion
import React from 'react';
import DesktopWrapper from '../desktop-wrapper';
import ErrorModalContent from './error-modal-content';
import MobileDialog from '../mobile-dialog';
import MobileWrapper from '../mobile-wrapper';
import Modal from '../modal';
```

Reply:
ok